core/roboconf-agent/src/main/java/net/roboconf/agent/internal/Agent.java

```diff
@@ -82,7 +81,7 @@
 	 */
 	public Agent() {
 		this.logger = Logger.getLogger( getClass().getName());
-		this.messagingFactoryType = MessagingConstants.FACTORY_RABBIT_MQ;
+		this.messagingType = MessagingConstants.FACTORY_RABBIT_MQ;
```

Comment:
I thin you did not addressed it in your last commit.
There should be no default messaging set in the agent and in the DM.